Library/Formula/i2p.rb

```diff
@@ -1,8 +1,8 @@
 class I2p < Formula
-  desc "I2P is an anonymous overlay network - a network within a network"
+  desc "An anonymous overlay network - a network within a network"
   homepage "https://geti2p.net"
-  url "https://download.i2p2.de/releases/0.9.21/i2pinstall_0.9.21.jar"
-  sha256 "0238ffc6ea44099ef4fe6c20913cb4eec675c2760aea07dfe7d499addcc89cf2"
+  url "https://dl.dropboxusercontent.com/u/18621288/I2P/0.9.23/i2pinstall_0.9.23.jar"
```

Comment:
Why aren't we using the `https://download.i2p2.de/releases/0.9.23/i2pinstall_0.9.23.jar` URL here? Seems to work okay.